lockbox-ios/Presenter/AutoLockSettingsPresenter.swift

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import RxSwift
+import RxCocoa
+import RxDataSources
+
+class AutoLockSettingsPresenter {
+    private var view: AutoLockSettingsProtocol
+    private var userInfoStore: UserInfoStore
+    private var routeActionHandler: RouteActionHandler
+    private var userInfoActionHandler: UserInfoActionHandler
+    private var disposeBag = DisposeBag()
+
+    var settings = Variable([AnimatableSectionModel(model: 0, items: [
```

Comment:
Nice use of `Variable`; however, it's being deprecated in future releases of RxSwift, so for future `Variable`s use `ReplaySubject` with a buffer size of 1.